packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.component.js

```diff
@@ -62,7 +58,12 @@ class RowCollapsiblePanel extends React.Component {
 						aria-label={get(rowData, 'header[0].label')}
 						style={style}
 					>
-						<PureCollapsiblePanel {...{ rowData, measure, onToggle: this.onToggle }} />
+						<CollapsiblePanel
+							onEntered={() => onMeasure(measure)}
+							onExited={() => onMeasure(measure)}
```

Comment:
could you please try onEntererd={measure} / same for onExited and remove onMeasure ?

Reply:
Done 👍